charts/backbeat-api/templates/deployment.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "backbeat-api.fullname" . }}
+  labels:
+    app: {{ template "backbeat-api.name" . }}
+    chart: {{ template "backbeat-api.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "backbeat-api.name" . }}
+        release: {{ .Release.Name }}
+    spec:
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          ports:
+            - name: backbeat-api
+              containerPort: 8900
+          env:
+            - name: ZOOKEEPER_AUTO_CREATE_NAMESPACE
+              value: "1"
+            - name: ZOOKEEPER_CONNECTION_STRING
+              value: "{{- printf "%s-zenko-zookeeper:2181" .Release.Name | trunc 63 | trimSuffix "-" -}}"
```

Comment:
I would rather let this error out at kub level than truncate the value, since it has zero chance to work if truncated (port or port+host will be incorrect so zookeeper will be unreachable) and the error may be harder to diagnose in such case (to be confirmed as I don't know what will actually happen).